### DISCUSSION TITLE
Feature request

### DISCUSSION THREAD

**Discussion by RaketenZonk** [OrcaSlicer/OrcaSlicer#309] [Ideas]
Hi!
First of all: thank you for this great fork!

I'm using BambuStudio SoftFever for my Sovol SV06 and I'm very happy so far. 
I have only one issue with the wizard:
With the instruction from BambuStudio I managed to add this printer including two different nozzle sizes, thumbnail, bed texture and conditional print settings and I'm able to chose them from the wizard, but it won't let me chose filaments and without any checked filaments I cannot finish the wizard. I've listed them with the correct names and paths (located in \profiles\Sovol\filament) in Sovol.json (located in \profiles) and they are also listed in the machine model files at "default materials". They also appear in the filament dropdown list under system presets after using my workaround which is to chose an additional printer, check a filament, finish the wizard and remove the other printer and filament again. After that I can use my preset filaments normally.

I'm also missing some minor features that would just be really nice to have. Maybe there is something you could do about in a future release?
I would like to
- disable g-code thumbnails. At the moment I'm setting this to 1x1 because 0 or 0x0 isn't valid
- chose my start-g-code to be the only start-code (except for pressure advance setting)
- chose to have minimalistic g-code without comments (except for custom g-code like start-, end-, layer change-, etc. g-code)
- use machine limits for g-code generation and calculating print duration but not to send them to the printer within start-g-code
- use "lay on face" for modifiers
- move objects (partially) below the print bed to print them from the plane where they hit the plate upwards without the need of cutting them

Thanks, 
Sophie

**Comment by boomer20021** [comment]
retraction test in to I think that will fix the string

**Reply by RaketenZonk** [reply]
Ah.. yes.. I don't think that's an answer to my request?!
